configs/setups/awicm3/awicm3.yaml

```
@@ -358,9 +357,9 @@ fesom:
                         nproc: 72
         choose_general.oasis_with_yac:
             True:
-                comp_command: mkdir -p build; cd build; cmake -DOIFS_COUPLED=ON -DFESOM_COUPLED=ON -DENABLE_OPENMP=ON -DOASIS_WITH_YAC=ON -DCMAKE_INSTALL_PREFIX=../  ..;   make install -j `nproc --all`
+                comp_command: mkdir -p build; cd build; cmake -DOIFS_COUPLED=ON -DFESOM_COUPLED=ON -DENABLE_OPENMP=ON OPENMP_REPRODUCIBLE=ON -DDISABLE_MULTITHREADING=OFF -DOASIS_WITH_YAC=ON -DCMAKE_INSTALL_PREFIX=../  ..;   make install -j `nproc --all`
```

```suggestion
                comp_command: mkdir -p build; cd build; cmake -DOIFS_COUPLED=ON -DFESOM_COUPLED=ON -DENABLE_OPENMP=ON -DOPENMP_REPRODUCIBLE=ON -DDISABLE_MULTITHREADING=OFF -DOASIS_WITH_YAC=ON -DCMAKE_INSTALL_PREFIX=../  ..;   make install -j `nproc --all`
```

Isn't the `OPENMP_REPRODUCIBLE=ON` missing a preceeding `-D`?

---

In the moment I set `-DOPENMP_REPRODUCIBLE` I start to have failing FESOM compilation, at least for awicm3-v3.2 (FESOM 2.5) with errors referring to the `!OMP_ORDERED` pragma: `/work/ab0246/a270152/testing/comp/awicm3/awicm3-v3.2/comp.out`